py3status/modules/bluetooth.py

```diff
@@ -25,6 +26,8 @@
 
 import dbus
 
+DEFAULT_FORMAT = 'BT: [{format_device}|\?show OFF]'
```

Comment:
Can we change this to `BT[: {format_device}]`? It shouldn't affect users with existing config.